securedrop/source_templates/generate.html

```diff
@@ -17,17 +14,19 @@ <h2 id="codename-heading" class="visually-hidden">{{ gettext('Codename') }}</h2>
 </section>
 
 <p class="explanation">
-  {{ gettext('Because we do not track users of our <b>SecureDrop</b>
- service, in future visits, using this codename will be the only way we have to communicate with you should we have
- questions or are interested in additional information. Unlike passwords, there is no way to retrieve a lost codename.') }}
+  {{ gettext('You will need this codename to log into our SecureDrop later.') }}
```

Comment:
This is not exactly as workshopped, but per chat with Kev, "access" seemed to retain some ambiguity, and we do use the term "log in" on the index page, so it seemed appropriate to use it here as well.

Reply:
Couple of potential tweaks:

1) the explanation text (first two lines) and the ordered list have different text colors - it's a little jarring. It might be better for the text to be styled consistently.
2) Looking at it in the page, it doesn't actually explicitly state that the codename displayed is the user's. It might be better to put both explanatory lines over the codename like so:

```
A codename in SecureDrop functions as both your username and your password.
You will need this codename to log into our SecureDrop later:

    [ lock] purple monkey dishwasher oil horse etc.

1. Keep it secret. Do not share it with anyone.
2. Keep it safe. There is no account recovery option.
```

Reply:
I agree. I've reorganized accordingly, and also made the list unnumbered.